tests/queries/0_stateless/01155_rename_move_materialized_view.sql

```diff
@@ -65,7 +65,9 @@ SELECT dictGet('test_01155_ordinary.dict', 'x', 'after renaming database');
 SELECT database, substr(name, 1, 10) FROM system.tables WHERE database like 'test_01155_%';
 
 -- Move tables back
+SET check_table_dependencies=0; -- Otherwise we'll get error "test_01155_atomic.dict depends on test_01155_ordinary.dist" in the next line.
```

Comment:
The comment is incorrect because there's no table `test_01155_atomic.dict` when we rename `test_01155_ordinary` to `test_01155_atomic`

Reply:
The comment should be

```
Otherwise we'll get error "test_01155_ordinary.dict depends on test_01155_ordinary.dist" in the next line.
```

I'll change that in my next PR.